samples/java-valueentity-customer-registry/src/main/java/customer/Main.java

```diff
@@ -27,6 +27,18 @@ public static AkkaServerless createAkkaServerless() {
                     "customerByName",
                     CustomerDomain.getDescriptor())
             // end::register[]
+            .registerView(
+                    CustomerViewModel.getDescriptor().findServiceByName("CustomerByEmail"),
+                    "customerByEmail",
+                    CustomerDomain.getDescriptor())
+            .registerView(
+                    CustomerViewModel.getDescriptor().findServiceByName("CustomerSummaryByName"),
+                    "customerSummaryByName",
+                    CustomerDomain.getDescriptor())
+            .registerView(
+                    CustomerViewModel.getDescriptor().findServiceByName("CustomersResponseByName"),
+                    "customersResponseByName",
+                    CustomerDomain.getDescriptor())
             .register(CustomerValueEntityProvider.of(CustomerValueEntity::new))
             .register(CustomerActionProvider.of(CustomerActionImpl::new));
```

Comment:
we could change this to `AkkaServerlessFactory.withComponents`

Reply:
I'll have a look, thanks.

Reply:
Done.